tests/ClientFunctionalTest.php

```diff
@@ -783,4 +783,33 @@ public function testPatchJobStatusRejected(): void
                 ->setDesiredStatus($createdJob->getDesiredStatus())
         );
     }
+
+    public function testGetJobsDurationSum(): void
+    {
+        $duration = 2;
+        $client = $this->getClient();
+        $jobFactory = $client->getJobFactory();
+
+        $job = $jobFactory->createNewJob([
+            '#tokenString' => getenv('TEST_STORAGE_API_TOKEN'),
+            'configId' => '12345',
+            'componentId' => 'keboola.ex-google-drive',
+            'mode' => 'run',
+        ]);
+
+        $job = $client->createJob($job);
+
+        $durationSum = $client->getJobsDurationSum($job->getProjectId());
+
+        $client->updateJob($jobFactory->modifyJob(
+            $job,
+            ['status' => JobFactory::STATUS_PROCESSING]
+        ));
+
+        sleep($duration);
+
+        $client->postJobResult($job->getId(), JobFactory::STATUS_SUCCESS, new JobResult());
+
+        self::assertSame($durationSum + $duration, $client->getJobsDurationSum($job->getProjectId()));
```

Comment:
```suggestion
        self::assertSame(0, $durationSum);
        self::assertSame($duration, $client->getJobsDurationSum($job->getProjectId()));
```
(jestli jsem to teda správně pochopil), protože na to pořád hledím a koukám proč by se to mělo sčítat

Reply:
Jsem to mel hlavne kvuli lokalnimu vyvoji, kdy tam mam i predchozi joby.
Pri buildu se vlastne ta DB initne pokazdy znovu, tak bych to mohl vyhodit.

Reply:
Ajo, už to chápu, no tak tam radši nech, ono by to pak mohlo být nestabilní pokud se testy nespustí ve stejným pořadí